src/main/java/ca/on/oicr/pineryreports/util/GeneralUtils.java

```diff
@@ -0,0 +1,22 @@
+package ca.on.oicr.pineryreports.util;
+
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.TemporalAccessor;
+import java.util.function.Predicate;
+
+import ca.on.oicr.ws.dto.RunDto;
+
+public class GeneralUtils {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)